Gemfile

```diff
@@ -72,12 +78,15 @@ gem 'sorcery'
 # gem 'mini_magick'           # => image manipulation
 # gem 'money-rails', '~>1'    # => money manipulation
 # gem 'pg_search'             # => pg-based search
+# gem 'phi_attrs'             # => phi logging and access control
 # gem 'pretender'             # => user impersonation
 # gem 'ranked-model'          # => sorting for models
 # gem 'ransack'               # => ruby-based search
 # gem 'responders'            # => automated responders
 # gem 'rollbar'               # => rollbar reporting
 # gem 'safely_block'          # => safely execute blocks
+# gem 'scenic'                # => save views in schema.rb
+# gem 'scenic-mysql_adapter'  # => scenic if using mysql
```

Comment:
Maybe a note here that scenic includes postgresql by default, and adapters are needed for other dbms?